pkg/util/ipset/ipset.go

```diff
@@ -322,4 +322,20 @@ func validatePortRange(portRange string) bool {
 	return true
 }
 
+// IsNotFoundError returns true if the error indicates "not found".  It parses
+// the error string looking for known values, which is imperfect but works in
+// practice.
```

Comment:
is there a reference to how these strings can be obtained in the future if something changes?

Reply:
xref: 

https://github.com/Olipro/ipset/blob/master/lib/errcode.c#L32-L33

https://github.com/Olipro/ipset/blob/master/lib/parse.c#L1904

https://github.com/Olipro/ipset/blob/master/lib/parse.c#L1925

Reply:
I added these reference now, PTAL.